shub/utils.py

```diff
@@ -100,7 +100,9 @@ def patch_sys_executable():
             try:
                 # Will raise NotFoundException if no Python installation found
                 py_exe = find_exe('python')
-                if '2.' not in get_cmd_output([py_exe, '--version']):
+                py_ver = subprocess.check_output([py_exe, '--version'],
+                                                 stderr=subprocess.STDOUT)
+                if '2.' not in py_ver:
```

Comment:
I'm a little afraid of unnecessarily excluding non-CPython implementations. E.g. what if PyPy outputs "PyPy 2.7.11". But then again I guess it doesn't, it is the language of the version and not the interpreter after all.